Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.ps1

```diff
@@ -1760,6 +1764,22 @@ function SetJunkRulesCommand([ExchangeOnlinePowershellV3Client]$client, [hashtab
     return $human_readable, $entry_context, $raw_response
 }
 
+function  CreateAddAndRemoveSections([string[]]$items_to_add, [string[]]$items_to_remove ){
```

Comment:
what about a case where theyre both empty?

Reply:
In that case it will be empty and not populated (already tested)